File: app/src/main/java/com/nevbg/simple/base/CustomBaseActivity.java
package com.nevbg.simple.base;

import android.os.Bundle;

import com.nevbg.simple.common.MyApplication;

import cn.sft.base.activity.BaseParseHelperActivity;

/**
 * Created by Administrator on 2018/3/20.
 */

public class CustomBaseActivity extends BaseParseHelperActivity {

    protected MyApplication app;

    @Override
    protected void onCreate(Bundle bundle) {
        if (app == null) {
            app = MyApplication.getInstance();
        }
        super.onCreate(bundle);
    }

    @Override
    protected void afterRestoreInstanceState(Bundle bundle) {

    }
}

File: app/src/main/java/com/nevbg/simple/base/CustomBaseActivity.kt
package com.nevbg.simple.base

import com.wxt.library.base.activity.BaseParseHelperActivity

/**
 * Created by Administrator on 2018/3/20.
 */

open class CustomBaseActivity : BaseParseHelperActivity() {

    /**
     * 标题栏标题是否居中
     * return true:居中，false：左对齐
     */
    override fun isTitleCenter() = true

    /**
     * 标题栏下方是否有横线（1px）
     * return true:有，false：没有
     */
    override fun isShowDownLine() = false

    /**
     * 标题栏是否显示导航按钮
     * return true:有，false：没有
     */
    override fun isShowNavigationBtn() = true
}
